### PR TITLE
redirect user to a Thank you page after report

### PR DESCRIPTION
**Pull request by KaitCrawford**
Added a static page, a url and a view

### CODE REVIEW

File: tuneme/templates/comments/report_response.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+</div>
+
+  <h4 class="left">Thanks!</h4>
+  
+<div class="block {{article.get_parent_section.get_effective_extra_style_hints}}">
+  <div class="article-copy">
+    <p><strong>This comment has been reported.</strong>
+    It will be reviewed by the Tune Me team 
+    and may be removed if it breaks our Platform rules</p>
+  </div>
```
Comment (KaitCrawford) [inline tuneme/templates/comments/report_response.html:16]: The above text should really be in the locale file or wagtail or something. @miltontony please advise

**Comment by KaitCrawford** [comment]
@miltontony @smn @saeedmb8 Ready for review once my comment has been addressed

File: tuneme/templates/comments/report_response.html (hunk 1 above)
Comment (miltontony) [inline tuneme/templates/comments/report_response.html:16]: @KaitCrawford `{% trans %}` for both those sentences.

**Comment by KaitCrawford** [comment]
@miltontony @smn @saeedmb8 Ready for review

File: tuneme/urls.py [hunk 2]
```diff
@@ -36,6 +36,7 @@
     ),
     url(r'^profiles/', include('molo.profiles.urls',
                                namespace='molo.profiles')),
+    url(r'^comments/reported/(\d+)', report_response, name='report_response'),
```
Comment (miltontony) [inline tuneme/urls.py]: I think you need to name this like so `url(r'^comments/reported/(?P<comment_pk>\d+)/$'`

**Comment by KaitCrawford** [comment]
@miton after I've addressed your note in urls.py are you happy to :+1: ?

**Comment by miltontony** [comment]
:+1:

**Pull request merged**